tests/sanity/ignore-2.14.txt

```diff
@@ -0,0 +1,64 @@
+plugins/inventory/hcloud.py validate-modules:illegal-future-imports
+
```

Comment:
Why is this line missing in comparison to `2.13`?

```suggestion
plugins/inventory/hcloud.py validate-modules:import-before-documentation
```

Reply:
Because the error seem not to show up in 2.14, and the CI fails if an ignore line is unused...